src/styles/styles.js

```diff
@@ -347,19 +347,27 @@ const styles = {
         width: variables.componentSizeNormal,
     },
 
-    textInput: {
+    textInputContainer: {
```

Comment:
Can you please put this new style below the `textInput` so the diff is easier to read?

Reply:
Sure